client/src/containers/TeamPage/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from 'react';
+import { Card, Grid, Message, Icon, Header, Image, Button } from 'semantic-ui-react';
+import { Link } from 'react-router-dom';
+import styles from './styles.module.scss';
+import { teamMembers } from 'services/team.service';
+import worksImg from 'assets/images/team-page-works.jpg';
+import linksImg from 'assets/images/team-page-links.jpg';
+
+export const TeamPage: React.FC = () => {
```

Comment:
Create separate components for small blocks from this page

Reply:
fixed